pkg/util/crypto.go

```diff
@@ -18,6 +18,7 @@ const (
 	DefaultRSABits = 2048
 	// For now the ClusterControlPlane API doesn't support rotating certificate. We set long valid time to avoid certificate expiration.
 	DefaultValidDays          = 3650
```

Comment:
We can add a DefaultValidDaysWithRotation and set it to 1 year. When NSX is >= 4.2, we can use DefaultValidDaysWithRotation for generating the cert, otherwise, use DefaultValidDays .

Reply:
Fixed. Currently we use `NSXCheckVersionForNSXServiceAccountRestore`(>=4.1.2), and added `TODO` marks on this part.

Reply:
Update the NSX version requirement to >=4.1.3